vault/identity_store_injector_testonly.go

```diff
@@ -359,16 +359,15 @@ func (i *IdentityStore) createDuplicateEntityAliases() framework.OperationFunc {
 			flags.Count = 2
 		}
 
-		ids, bucketIds, err := i.CreateDuplicateEntityAliasesInStorage(ctx, flags)
+		ids, err := i.CreateDuplicateEntityAliasesInStorage(ctx, flags)
```

Comment:
Note that none of our tests in the code base use bucket_keys from this response any more so this seems reasonable.